services/ocr/utils/recogniseTextOcrSpace.js

```diff
@@ -41,7 +41,7 @@ export const recogniseTextOcrSpace = async (fileName, language, logger) => {
             || error.message === 'socket hang up'
             || error.message === 'connect ETIMEDOUT'
             || error.message === 'Proxy connection timed out'
-            || error.message === 'Socks5 proxy rejected connection - TTLExpired'
+            || error.message.startsWith('Socks5 proxy rejected connection')
```

Comment:
Are they all socks5? If not, maybe matchning for `proxy rejected connection` would be better?